spec/models/pdc_serialization/datacite_spec.rb

```diff
@@ -247,6 +248,27 @@
     end
   end
 
+  describe "resource JSON to Datacite XML" do
+    fixtures_dir = "spec/fixtures/resource-to-datacite"
+    Dir.glob("#{fixtures_dir}/*.resource.yaml").each do |resource_path|
+      it "handles #{resource_path}" do
+        resource_json = YAML.load_file(resource_path).to_json
+        resource = PDCMetadata::Resource.new_from_json(resource_json)
+        datacite_xml = resource.to_xml
+
+        datacite_path = resource_path.gsub(".resource.yaml", ".datacite.xml")
+        datacite_xml_expected = File.read(datacite_path)
+        expect(datacite_xml).to be_equivalent_to(datacite_xml_expected)
+      end
+    end
+
+    it "does not contain stray files" do
+      Dir.glob("#{fixtures_dir}/*").each do |path|
+        expect(path).to match(/\.resource\.yaml$|\.datacite\.xml$/)
```

Comment:
Where would these stray files come from??

Reply:
From me, perhaps: In the past, I have found that having directories of fixtures that are turned into tests can be useful. But at least once I've tripped up because I added a fixture that didn't match the expected glob pattern, and so it wasn't testing against an example I had provided.

This prevents that.

Reply:
Could you add a comment in the test to say this is why... Maybe something like If this test fails you have an example file that is not being tested...